### ISSUE TITLE
SelectButton: aria-disabled value incorrectly set when option is disabled preventing accurate screen reader 

### ISSUE THREAD

**Issue by ghost** [primefaces/primeng#15907]
### Describe the bug

I noticed while testing with NVDA on the SelectButton Component that disabled options were not reading out as disabled. Looking at the DOM I saw that the 'aria-disabled' attribute was there, but the value was set to 'disabled'. This should be set to 'aria-disabled:="true"' in order to provide the appropriate read out.

It seems optionDisabled may be part of the issue?

### Environment

Windows 11
Using Edge browser
NVDA screen reader
Angular 17

### Reproducer

https://stackblitz.com/edit/github-qow2ft?file=src%2Fapp%2Fapp.component.html

### Angular version

17

### PrimeNG version

17.18.1

### Build / Runtime

Angular CLI App

### Language

TypeScript

### Node version (for AoT issues node --version)

18

### Steps to reproduce the behavior

Without NVDA
- Notice you can tab to disabled options in the [repro env](https://stackblitz.com/edit/github-qow2ft?file=src%2Fapp%2Fapp.component.html)

- Download NVDA if you don't have it [download](https://www.nvaccess.org/download/)
- Go to [repro env](https://stackblitz.com/edit/github-qow2ft?file=src%2Fapp%2Fapp.component.html) and start 
Start NVDA and then use your mouse to click on the repro example ANYWHERE
Now use tab or shift + tab to navigate the focus to the options, when focused on a disabled option notice the read out does not include the disabled adjective,
- inspect the disabled option, see the aria-disabled attribute is set to disabled

### Expected behavior

disabled options will have aria-disabled= "true" and enabled options would have aria-disabled="false"
Alternately, removing the tab index from disabled options would also suffice a11y

**Comment by mertsincan** [comment]
Hi,

So sorry for the delayed response! Improvements have been made to many components recently, both in terms of performance and enhancement. Therefore, this improvement may have been developed in another issue ticket without realizing it. You can check this in the documentation. If there is no improvement on this, can you open a new issue so we can include it in our roadmap?

Thanks a lot for your understanding!
Best Regards,
